packages/nx-go/src/generators/application/generator.ts

```diff
@@ -60,11 +60,7 @@ export default async function applicationGenerator(
   generateFiles(tree, join(__dirname, 'files'), options.projectRoot, options);
 
   if (isGoWorkspace(tree)) {
-    createGoMod(
-      tree,
-      `${options.npmScope}/${options.moduleName}`,
-      options.projectRoot
-    );
+    createGoMod(tree, options.projectRoot, options.projectRoot);
```

Comment:
To do not to break other calls in the project, for the moment I've set the same value in both parameters.

Reply:
Ok, I will review tonight ☺️